packages/amplication-data-service-generator-runner/Dockerfile

```diff
@@ -1,6 +1,13 @@
+ARG GIT_REF
+
 FROM node:19.1-alpine AS base
+ARG GIT_REF
+
 RUN apk add g++ make python3
 
 WORKDIR /workspace/packages/amplication-data-service-generator-runner
 COPY ./dist/packages/amplication-data-service-generator-runner .
-RUN npm i --omit-dev
\ No newline at end of file
+RUN npm i --omit-dev
+
+ENV DSG_VERSION=${GIT_REF}
+RUN echo ${GIT_REF}
```

Comment:
OK, now I see how the `INPUT_BUILD_ARGS: GIT_REF=${{ github.ref }}` is related. But can you explain? :)

Reply:
I'll use a diagram to explain this but in a nutshell, `ARG` are used at build time and `ENV` is used at runtime:
![image](https://user-images.githubusercontent.com/2861984/216282466-e4375bc5-9aa5-43e3-ac9e-fc6205be0d7f.png)